lib/mongoidable/document_extensions.rb

```diff
@@ -8,16 +8,5 @@ module DocumentExtensions
     included do
```

Comment:
Remove model dump/load. No longer needed since we do not directly cache the rules.